package.json

```diff
@@ -38,12 +39,14 @@
     "@types/node-fetch": "^2.5.7",
     "@typescript-eslint/eslint-plugin": "^4.3.0",
     "@typescript-eslint/parser": "^4.3.0",
+    "assemblyscript": "^0.16.1",
```

Comment:
There might be issues here as this uses a different version of Asc then The graph, and there may be incompatibilities. Any chance we can use the same version The graph does (`https://github.com/AssemblyScript/assemblyscript#36040d5`)?